source/solutions-library/claim-management.txt

```diff
@@ -87,67 +91,121 @@ context.
 Data Model Approach
 -------------------
 
-The “claim” collection contains documents including a number of fields
-related to the claim. In particular, we are interested in the
-``claimDescription`` field, which we vectorize and add to the document
-as ``claimDescriptionEmbedding``. This embedding is then indexed and
-used to retrieve documents associated with the user prompt.
-
-.. code-block:: json
-   :emphasize-lines: 4, 10
-   :copyable: true
-
-   {
-      _id: ObjectId('64d39175e65'),
-      customerID: "c113",
-      claimDescription: "A motorist driving...",
-      damageDescription: "Front-ends of both...",
-      lossAmount: 1250,
-      photo: "image_65.jpg"
-      claimClosedDate: "2024-02-03",
-      coverages: Array(2),
-      claimDescriptionEmbedding: [0.3, 0.6, ..., 11.2]
-   }
-
-Building the Solution
----------------------
-
-The instructions to build the demo are included in the README of `this
-Github repo <https://github.com/mongodb-industry-solutions/RAG-Insurance>`__.
+In the demo solution, the data model is a simplified design that emulates
+real-world insurance claim data. The approach leverages MongoDB's flexible
+document model to handle the diverse data structure that stores embeddings
+alongside their related document.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 8 10 20
```

Comment:
while i really appreciate finding different ways to display content (this is awesome!), we do want to try not to include large code blocks within tables because it kinda messes with SEO stuff. I might recommend breaking this table down :( sad!!!